core/src/test/java/org/bitcoinj/core/AbstractFullPrunedBlockChainTest.java

```diff
@@ -234,13 +234,12 @@ public void testFinalizedBlocks() throws Exception {
     
     @Test
     public void testFirst100KBlocks() throws Exception {
-        Context context = new Context(MAINNET);
         File blockFile = new File(getClass().getResource("first-100k-blocks.dat").getFile());
         BlockFileLoader loader = new BlockFileLoader(MAINNET, Arrays.asList(blockFile));
         
         store = createStore(MAINNET, 10);
         resetStore(store);
-        chain = new FullPrunedBlockChain(context, store);
+        chain = new FullPrunedBlockChain(MAINNET, store);
```

Comment:
This comment isn't related to this PR, but I wanted to note there is potential confusion in unit tests between some of these `NetworkParameters` constants and `BitcoinNetwork` `enum` instances. As long as we avoid static imports of the `BitcoinNetwork` instances we should be fine.